pkg/policy/portmap.go

```diff
@@ -44,14 +46,18 @@ type NamedPortMap map[string]PortProto
 // PortProtoSet is a set of unique PortProto values.
 type PortProtoSet map[PortProto]struct{}
 
-// Equal returns true if the PortProtoSets are equal.
-func (pps PortProtoSet) Equal(other PortProtoSet) bool {
-	if len(pps) != len(other) {
+// DeepEqual returns true if the PortProtoSets are equal.
+func (pps PortProtoSet) DeepEqual(other *PortProtoSet) bool {
```

Comment:
Why this change when there is nothing that is autogenerated here? You want the call sites be written as if `DeepEqual()` used autogenerated code?

Reply:
Yes. The autogenerated structures that use this field will call DeepEqual for this structure so I end up renaming this function as well.